akka-http2-support/src/test/scala/akka/http/impl/engine/http2/WithPriorKnowledgeSpec.scala

```diff
@@ -6,28 +6,21 @@ package akka.http.impl.engine.http2
 
 import java.util.Base64
 
-import akka.http.impl.util.WithLogCapturing
+import akka.http.impl.util.AkkaSpecWithMaterializer
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.model.{ HttpProtocols, HttpRequest, HttpResponse, StatusCodes }
-import akka.stream.{ ActorMaterializer, OverflowStrategy }
-import akka.stream.scaladsl.{ Keep, Sink, SinkQueue, Source, Tcp }
-import akka.testkit.AkkaSpec
+import akka.stream.OverflowStrategy
+import akka.stream.scaladsl.{ Keep, Source, Tcp }
 import akka.util.ByteString
 
 import scala.concurrent.Future
 
-class WithPriorKnowledgeSpec extends AkkaSpec("""
-    akka.loglevel = debug
-    akka.loggers = ["akka.http.impl.util.SilenceAllTestEventListener"]
+class WithPriorKnowledgeSpec extends AkkaSpecWithMaterializer("""
```

Comment:
@viktorklang summoned you for review in case you are interested because you contributed this test here.

Reply:
Thanks, @jrudolph—I trust your judgement on this PR!